### PR TITLE
HSC-4: Add 'fr' translations

### PR DESCRIPTION
**Pull request by chinzou**
https://mekomsolutions.atlassian.net/projects/HSC/issues/HSC-4

### CODE REVIEW

File: configuration/openmrs/i18n/clinical/locale_fr.json [hunk 1]
```diff
@@ -61,7 +61,7 @@
   "DISCHARGE_SUMMARY_TREATMENTS_KEY": "Médicaments",
   "OBSERVATIONS_BOARD_LABEL_KEY": "Observations",
   "MEDICATIONS_BOARD_LABEL_KEY": "Médicaments",
-  "ORDERS_BOARD_LABEL_KEY": "Examens",
+  "ORDERS_BOARD_LABEL_KEY": "Ordonnance",
```
Comment (rbuisson) [inline configuration/openmrs/i18n/clinical/locale_fr.json]: Missing an "s" ?

File: configuration/openmrs/i18n/clinical/locale_fr.json [hunk 2]
```diff
@@ -289,8 +288,6 @@
   "ALL_LAB_RESULTS_LABEL": "Tous les résultats de laboratoire",
   "ACCESSION_AT_LABEL": "Adhésion à",
   "SAMPLE_AT_LABEL": "Échantillon à",
-  "CLINICAL_INITIAL": "Initial",
-  "CLINICAL_CURRENT": "En cours",
```
Comment (rbuisson) [inline configuration/openmrs/i18n/clinical/locale_fr.json]: Not needed?

File: configuration/openmrs/i18n/registration/locale_fr.json [hunk 3]
```diff
@@ -100,6 +100,7 @@
 
   "OBS_ABNORMAL_TRANSLATE_KEY": "Anormal",
   "TRIAGE_INFORMATION": "Triage",
-  "VITAL_SIGNS": "Constantes"
-
+  "VITAL_SIGNS": "signes vitaux",
+  "Enter Phone Number":"Entrer numéro de Téléphone",
```
Comment (rbuisson) [inline configuration/openmrs/i18n/registration/locale_fr.json]: So, should we not replace this by an more 'key-like' label?

Comment (chinzou) [reply configuration/openmrs/i18n/registration/locale_fr.json]: yes, sure

File: configuration/openmrs/i18n/clinical/locale_fr.json (hunk 1 above)
Comment (chinzou) [reply configuration/openmrs/i18n/clinical/locale_fr.json]: yes, thanks

File: configuration/openmrs/i18n/clinical/locale_fr.json (hunk 2 above)
Comment (chinzou) [reply configuration/openmrs/i18n/clinical/locale_fr.json]: these are duplicated in the same file

**Pull request merged**